db/migrations/20220630002212-add-github-client-secrets-to-github-server-apps.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+const tableName = "GitHubServerApps";
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		await queryInterface.addColumn(tableName, "gitHubClientSecret", {
+			type: Sequelize.TEXT,
+			allowNull: false
+		});
+		await queryInterface.addColumn(tableName, "webhookSecret", {
+			type: Sequelize.TEXT,
+			allowNull: false
+		});
+		await queryInterface.addColumn(tableName, "privateKey", {
+			type: Sequelize.TEXT,
+			allowNull: false
+		});
+		await queryInterface.removeColumn(tableName, "secrets");
```

Comment:
should we be explicit in the naming and call it `encryptedPrivateKey`, `encryptedWebhookSecret` and `encryptedGithubClientSecret`

Reply:
But, in the code we will say `app.encryptedWebhookSecret = "some-plain-text"`, wouldn't that be strange?

Reply:
we can change the variable name in the code as part of the model init :)

The code would stay the same, just the DB column would have a different name

Reply:
you mean just change the db column name? Ah, I am cool with that.

Reply:
implemented